lib/fluent/plugin/out_kafka_buffered.rb

```diff
@@ -63,6 +63,7 @@ class Fluent::KafkaOutputBuffered < Fluent::BufferedOutput
 Supported codecs: (gzip|snappy)
 DESC
   config_param :max_send_limit_bytes, :size, :default => nil
+  config_param :skip_kafka_delivery_failed, :bool, :default => false
```

Comment:
Logging warning message in `configure` is better when this option is true.
"This option discards events which cause delivery failure, e.g. invalid topic or something. If this is unexpected, you need to check your configuration or data"